mautic/mautic
# do not show blank chart when no data is found

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When you create a lifecycle widget, device granularity widget, an email granularity widget or a created contacts widget, nothing is shown when there is no data found for the date range used to filter. This PR shows a no data for this report message

#### Steps to test this PR:
1. Apply this PR
2. create a lifecycle widget, device granularity widget, email granularity widget and a created contacts widget, if there is no data to show a 'no data for this report' should be shown

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. create a lifecycle widget, device granularity widget, email granularity widget and a created contacts widget an empty space is shown
